react/src/components/Table/ColumnFilter.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Flex, Input } from '../index';
+
+interface ColumnFilterProps {
+    setFilter: (columnId: string, filterValue: any) => void;
+    columnId: string;
+}
+
+const SearchInput = styled(Input)`
+    min-height: 30px;
+`;
+
+export const ColumnFilter: React.FC<ColumnFilterProps> = ({ setFilter, columnId }) => {
+    const [value, setValue] = useState('');
```

Comment:
Can you make this component controlled so that it takes `state.filters` as a prop rather than storing the filter value in local state?